### PR TITLE
here is my nifity mode of Wipro

### PR DESCRIPTION
**Pull request by PRIYANSHU2026**
I got 99% accuracy

### CODE REVIEW

**Comment by Amarta113** [comment]
@PRIYANSHU2026 Does achieving 99% accuracy suggest potential overfitting?

**Comment by PRIYANSHU2026** [comment]
@Amarta113  yes if u are getting 99.9 is coming underfitting but if you get 100% then it is overfitting

**Comment by Amarta113** [comment]
> @Amarta113 yes if u are getting 99.9 is coming underfitting but if you get 100% then it is overfitting

How 99.9 is underfitting? Do you mean it is coming overfitting?

**Comment by PRIYANSHU2026** [comment]
> > @Amarta113 yes if u are getting 99.9 is coming underfitting but if you get 100% then it is overfitting
> 
> How 99.9 is underfitting? Do you mean it is coming overfitting?

it is depend upon Data set but my one is very large so it is predicting well for Small data set it is overfitting understood

**Comment by Amarta113** [comment]
> > > @Amarta113 yes if u are getting 99.9 is coming underfitting but if you get 100% then it is overfitting
> > 
> > 
> > How 99.9 is underfitting? Do you mean it is coming overfitting?
> 
> it is depend upon Data set but my one is very large so it is predicting well for Small data set it is overfitting understood

yes now I understood

**Review by Akshat111111 — COMMENTED** [review]
you have done analysis of adani port, and written wipro??

**Comment by PRIYANSHU2026** [comment]
May be I have done mistak

**Comment by PRIYANSHU2026** [comment]
sir from my I did that much

**Comment by EwoutH** [comment]
Have you fully separated your train, test and validation training sets?

Also consider doing cross validation.

**Comment by Akshat111111** [comment]
> Have you fully separated your train, test and validation training sets?
> 
> Also consider doing cross validation.

Yeah @EwoutH is correct, kindly consider it

**Comment by PRIYANSHU2026** [comment]
sure sir

**Review by Akshat111111 — COMMENTED** [review]
The wipro analysis folder still exist with adani analysis
